sensor/temperature.h

```diff
@@ -0,0 +1,14 @@
+#ifndef __TEMPERATURE_H_
+#define __TEMPERATURE_H_
+#include <vector>
+
+#define TEMPERATURE_MIN_VALUE  (-50)
+#define TEMPERATURE_MAX_VALUE  (150)
+
+using namespace std;
+
+vector<int> generateTemperatureValues(int teperatureMaxValue, int teperatureMinValue, int numberOfReadings);
```

Comment:
Please correct the spell of the variable, so that it would be more meaning while reading.

Reply:
changes done.